lib/ui/views/accounts/account_list.dart

```diff
@@ -330,6 +330,16 @@ class _AccountListItem extends ConsumerWidget {
                     .read(AccountProviders.accounts.notifier)
                     .selectAccount(account);
               }
+              ref
```

Comment:
We could create an `refresh` method that refreshed all those data.
That way, we would call `getLastTransactionAddress`only once.